client/layout/guided-tours/config-elements/step.js

```diff
@@ -60,6 +60,12 @@ export default class Step extends Component {
 
 	static contextTypes = contextTypes;
 
+	/**
+	 * Flag to determine if we're repositioning the Step dialog
+	 * @type {boolean} True if the Step dialog is being repositioned.
+	 */
+	animating = false;
```

Comment:
Maybe rename this one to `isUpdatingPosition` or something similar. Apart from that this looks good to go to me. Thanks!